src/legacy/core_plugins/kibana/public/dashboard/panel/panel_utils.test.js

```diff
@@ -33,8 +37,24 @@ test('parseVersion', () => {
 
 test('convertPanelDataPre_6_1 gives supplies width and height when missing', () => {
   const panelData = [
-    { col: 3, id: 'foo1', row: 1, type: 'visualization', panelIndex: 1 },
-    { col: 3, id: 'foo2', row: 1, size_x: 3, size_y: 2, type: 'visualization', panelIndex: 2 }
+    {
+      col: 3,
+      id: 'foo1',
+      row: 1,
+      type: 'visualization',
+      panelIndex: '1',
+      gridData: createPanelState,
+    },
+    {
+      col: 3,
+      id: 'foo2',
+      row: 1,
+      size_x: 3,
+      size_y: 2,
+      type: 'visualization',
+      panelIndex: '2',
```

Comment:
Since we've learned about `panelIndex` potentially being a number in older versions, we may want to keep these as `number`s as they were originally

Reply:
d3ad4ef should reflect this change being implemented